README.md

```diff
@@ -32,6 +32,41 @@ To add or modify point data, change the apertiumPairs.json file.
 
 * world-110m.json -- Country data used for generating the globe
 
+## Scraping Data For Pairs:
+* First, run [this program](https://svn.code.sf.net/p/apertium/svn/trunk/apertium-tools/get_all_lang_pairs.py) in terminal.
+```
+$ python3 <path_to_file>/get_all_lang_pairs.py > pairs.txt
+```
+
+* This should create a new file called `pairs.txt` and fill it with a list of pairs. This step might take a while (5-10 minutes maybe). In the end, `pairs.txt` should look something like this:
+```
+{'lg1': 'af', 'lg2': 'nl', 'last_updated': '2016-07-23', 'created': '2007-09-24', 'direction': '<>', 'repo': 'trunk', 'stems': 6263}
+{'lg1': 'arg', 'lg2': 'cat', 'last_updated': '2017-10-27', 'created': '2016-01-04', 'direction': '<>', 'repo': 'trunk', 'stems': 24267}
+{'lg1': 'bel', 'lg2': 'rus', 'last_updated': '2017-08-03', 'created': '2016-03-08', 'direction': '<>', 'repo': 'trunk', 'stems': 48880}
+{'lg1': 'br', 'lg2': 'fr', 'last_updated': '2017-07-19', 'created': '2007-05-02', 'direction': '<>', 'repo': 'trunk', 'stems': 27998}
+{'lg1': 'ca', 'lg2': 'it', 'last_updated': '2017-07-03', 'created': '2009-07-17', 'direction': '<>', 'repo': 'trunk', 'stems': 9772}
+{'lg1': 'cat', 'lg2': 'srd', 'last_updated': '2017-10-28', 'created': '2010-10-28', 'direction': '<>', 'repo': 'trunk', 'stems': 36517}
+{'lg1': 'crh', 'lg2': 'tur', 'last_updated': '2017-11-22', 'created': '2017-04-07', 'direction': '<>', 'repo': 'trunk', 'stems': 7082}
+{'lg1': 'cy', 'lg2': 'en', 'last_updated': '2015-12-13', 'created': '2007-05-02', 'direction': '<>', 'repo': 'trunk', 'stems': 11608}
+...
+```
+
+* Run [this program](https://raw.githubusercontent.com/jonorthwash/Apertium-Global-PairViewer/master/formatpairs.py) to format it the `pairs.txt` file. This should generate a new file `formattedpairs.txt` that contains the formatted pairs. Note: you should be running `formatpairs.py` in the same directory as `pairs.txt`.
+
+For Mac:
+```
+curl https://raw.githubusercontent.com/jonorthwash/Apertium-Global-PairViewer/master/formatpairs.py -o formatpairs.py && python3 formatpairs.py
```

Comment:
I don't think this curl works. Also, if you're gonna use a code block, just put it in two lines instead of the `&&`

Reply:
@sushain97 I tested it. It should work. Note that https://raw.githubusercontent.com/jonorthwash/Apertium-Global-PairViewer/master/formatpairs.py actually doesn't exist yet.